editor/editor_properties.cpp

```diff
@@ -221,6 +227,13 @@ void EditorPropertyPath::_path_pressed() {
 	}
 
 	String full_path = get_edited_object()->get(get_edited_property());
+	if (full_path.is_rel_path()) {
+
+		if (!DirAccess::exists(full_path.get_base_dir())) {
+			DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+			da->make_dir_recursive(full_path.get_base_dir());
```

Comment:
You should then `memdelete(da)`, or better, use `DirAccessRef` instead of `DirAccess`.

Reply:
BTW, I agree with creating the folder when exporting, but why also create it here? Isn't this the function that runs whenever the path is changed? I don't think we should go create empty directories at this point, until the user confirms that they mean to use them by exporting.

Reply:
I create the path here due to believing that if a person is exporting the project from a shared repository, then it would make sense that if they pressed the path change button, it would create and bring them to the target folder. Not sure if there was a prompt asking the user if the directory exists, so I just created it there.
Will change DirAccess into DirAccessRef asap.